Supplements/Virgo/portalContext-security.xml

```diff
@@ -93,6 +114,11 @@
     </beans:bean>
     -->
 
+    <beans:bean id="loginKeycloakFilter" class="org.apromore.portal.security.LoginRedirectKeycloakFilter">
+        <beans:property name="keycloakLoginFormUrl"
+                        value="http://securityms.apromoresso.net:8282/sso/initiateLogin?fullReturnPath=&lt;full_return_path&gt;" />
```

Comment:
I feel like using context:property-placeholder to pull this hardcoded URL from site.cfg and having a ${} macro will eventually happen, but it's too much hassle until the config file can be moved from the bundle classpath to an external config file.  Since the URL does get repeated a few times, maybe it's worth defining as a property constant just inside portalContext-security.xml.

Reply:
Utilised a property constant, and referenced in both places [DONE].

Reply:
Agreed with Simon to leave as is, for this release [DONE].